charts/opentelemetry-collector/templates/_config.tpl

```diff
@@ -156,6 +156,13 @@ receivers:
         memory:
         disk:
         filesystem:
+          exclude_mount_points:
```

Comment:
I like your idea of making this optional, making something like

```yaml
hostMetrics:
    enabled: false
    excludeKubeletStatsFilesystems: false
```

or something like that.  A comment explaining when to use it would be good as well.

Reply:
@TylerHelmuth thanks. I'll add support for that structure.